### PR TITLE
Use API to fetch data

### PR DESCRIPTION
**Pull request by faisalakb**
In this work, applied the following features :100: 
- Getting data from API :+1: 
- Render API data into new component :+1: 
- Display all states on the screen : Success, Pending, Error :+1: 
- Setup React. :+1: 
- Setup Linters :+1:

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -33,25 +34,29 @@ const Calculator = () => {
...
+        </div>
+        <div className="zeroDiv">
+          <button type="button" onClick={() => setValue('0')} className="zero">0</button>
+          <button type="button" onClick={() => setValue('.')}>.</button>
+        </div>
       </div>
-    </div>
+
+      <DisplayApi />
+    </>
```
Comment (Qoosim) [inline src/components/Calculator.jsx]: - [ ] Kindly move your `DisplayApi` to the `App.js` component for better structure. `Calculator` is a separate component, `DisplayApi` is also a separate component, and  `DisplayApi` does not have anything to do inside the `Calculator` component. Kindly make it to look something similar to this. :point_down: 

```
import Calculator from "./components/Calculator"
import Quotes from "./components/Quotes"

function App() {
  return (
    <>
      {/* Rest of JSX */}
      <Calculator />
      <Quotes />
    </>
  )
}

export default App

```

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @faisalakb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Data fetched from the API correctly. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi @faisalakb 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 


## Highlights

The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by faisalakb** [comment]
Thanks for the review

**Pull request merged**